src/test/tests/rendering/scalable.py

```diff
@@ -358,25 +359,26 @@ def ForceSRMode(mode):
 
 DeleteAllPlots()
 
-TestSection("Stereo Modes")
+# this section causes a crash, comment out until fixed.
+#TestSection("Stereo Modes")
```

Comment:
I am concerned we'll completely forget about these being commented out.

I know we can skip specific test cases. Does that not get us around this crash?

I tested 2.13.3 SR+Red/Blue stereo manually on OSX and it works fine. Is the crash specific to 3.0.0?

Reply:
New ticket #3513 entered to address the problem with stereo mode.
Yes, this is now to 3.0.

Reply:
Ok, fair enough I guess. I also tested 3.0.0 (serial mode though because my freakin' machine doesn't know its own hostname well enough to launch a parallel engine right now) and SR+Red/Blue stereo worked in serial.